src/cmd/flux-job.c

```diff
@@ -145,6 +145,13 @@ static struct optparse_option list_inactive_opts[] =  {
     OPTPARSE_TABLE_END
 };
 
+static struct optparse_option list_ids_opts[] =  {
```

Comment:
Commit message:

> However, the flux job list-ids does not support anything to use it.

Suggestion: "the flux job list-ids command does not support any way to use it" ?